rst_files/mccall_model.rst

```diff
@@ -407,10 +395,10 @@ between successive iterates is below `tol`
             v_next[j] = max(stop_val, cont_val)
         end
 
-        error = maximum(abs.(v_next - v))
+        error = maximum(abs(a - b) for (a, b) in zip(v_next, v))
```

Comment:
I'm fine with this. But maybe this is the kind of terse, high-density syntax that @jlperla wants to avoid.

Reply:
That whole lectures needs to be re-written, what I might end up favoring is defining a helper function to make it simpler for most lectures... I will think about it.

Reply:
Yeah.  I think the key is that since so much will be rewritten (e.g. this particular example should be replaced with the fixedpoint library!)   it isn't worth doing too much refactoring right now (in either direction... changing the existing code beyond getting it running, or worrying about changes needed to get it running)

But. that said, I think that `error = maximum(abs(a - b) for (a, b) in zip(v_next, v))` is pretty tough and far from the math.  THis is the sort of thing where `error = norm(v_next - v)` is probably appropriate.  Even if it is slightly slower in general, it is mathematically clear.

Reply:
Yeah, no need to make actual code changes. Just thought I'd bring it up.